frontend/mgramseva/lib/providers/bill_generation_details_provider.dart

```diff
@@ -134,8 +134,7 @@ class BillGenerationProvider with ChangeNotifier {
       var reqDate = readDate.add(Duration(days: 1)).toLocal().toString();
       prevReadingDate = DateFormats.dateToTimeStamp(
           DateFormats.getFilteredDate(reqDate, dateFormat: 'dd/MM/yyyy'));
```

Comment:
Create a date formate constant class and read from it.

Reply:
DateFormats is already a static class,
which is accessed from egov\mGramseva-HRMS\punjab-mgramseva\frontend\mgramseva\lib\utils\date_formats.dart